packages/contracts/contracts/tokens/ERC20Token.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity 0.4.18;
+
+import "./Token_v2.sol";
+
+contract ERC20Token is Token_v2 {
+
+    function transfer(address _to, uint _value)
+        public
+        returns (bool) 
+    {
+        require(balances[msg.sender] >= _value && balances[_to] + _value >= balances[_to]); 
```

Comment:
doesn't everything in here need to be safe mathified

Reply:
It's an option, but this line checks for overflows anyways.